### ISSUE TITLE
Remove instance of "Esmail EL BoB"

### ISSUE THREAD

**Issue by MagicLike** [libreddit/libreddit-instances#81]
Esmail is actively forbidding members or supporters of the LGBTQIA+ community to use their services via a TOS document*.
This is absolutely queerphobic and extremely discriminating. Therefore I ask to remove their instance `libreddit.esmailelbob.xyz` / `libreddit.esmail5pdn24shtvieloeedh7ehz3nrwcdivnfhfcedl7gf4kwddhkqd.onion` from the list, to not further support their actions.

*See the commit here: https://github.com/EsmailELBoBDev2/upptime/commit/c46b9bfd03b211dff2b87737da2de33629685a5a

**Comment by AhmadDakhlallah** [comment]
This is awesome. We're punishing people for whatever they want to do in thier own instance because you don't like how they think.

We force our freedom over others so they can be less free because we need to be free?

If you disagree with the instance, don't use it. You have full freedom to do so. You're not accomplishing anything by hacking it and removing it. I mean other than being an oppressor who is forcing his "freedom" by taking others freedom.

Would you do the same if the instance is gils only or gay only or doctors only or some other groups only people? This one is straight people only. If that's not acceptable, you need to take down a lot more than this one. Many instances are specific to a group of people.

**Comment by xbdmHQ** [comment]
> This is awesome. We're punishing people for whatever they want to do in thier own instance because you don't like how they think.
> 
> We force our freedom over others so they can be less free because we need to be free?
> 
> If you disagree with the instance, don't use it. You have full freedom to do so. You're not accomplishing anything by hacking it and removing it. I mean other than being an oppressor who is forcing his "freedom" by taking others freedom.
> 
> Would you do the same if the instance is gils only or gay only or doctors only or some other groups only people? This one is straight people only. If that's not acceptable, you need to take down a lot more than this one. Many instances are specific to a group of people.

Not at all is not, some don't change the code sense it just a privacy frontend, we don't skid and block gays atleast on mines. discuss.whateveritworks.org, everyone is welcome and read the privacy policy on https://www.whateveritworks.org/privacy

**Comment by NoPlagiarism** [comment]
>  We're punishing people for whatever they want to do in thier own instance

pls, don't say that it like that. It sounds like they can even destroy their instance or publish malicious code there or even host porn instead of libreddit, cus yeah, it's their property. And they can do it, but libreddit should moderate their own instance list.

> We force our freedom over others so they can be less free because we need to be free?

Same logic: It's libreddit team's freeedom to moderate instance list in their way. You can agree, disagree, create your own instance list or still use instances that were removed from official list

**Comment by xbdmHQ** [comment]
Yeah, and not all instances are good some leave logs on, insert code, ban gays, etc... The only thing you should be doing is maintaining it, and helping others fight for privacy and turning off all logs tbh

**Comment by tuxayo** [comment]
Since the people here defending the instance care about privacy: how the hell does that respect privacy to discriminate on sexual orientation? Isn't that one of the most private things? Sure they collect nothing but that's still meddling in people intimacy for no good reason.

This list should have instances that are really open to the public unless they have a good reason, it's for widespread use. TOS banning «part of the LGBTQ+ community or support LGBTQ+ community» makes it not public instance. It's not like forbidding abuse or a political group that advocates against human rights. Like no nazis, no racists, etc.

It should not be required to users of this list to systematically check the TOS for confirming if **basic** human rights are respected. Sure they should be checked for more subtile stuff but there should be a baseline.



@xbdmHQ 
> Yeah, and not all instances are good

What's wrong with removing the explicitly bad ones? The point of libreddit shouldn't be to still face a bad online service.

> insert code

I hope an instance inserting very malicious code would get removed. Or even that would be ok?

> The only thing you should be doing is maintaining it, and helping others fight for privacy and turning off all logs tbh

And it's not a problem if people don't have a right to privacy because of their sexual orientation? Or for **not being hostile** (it’s passive support) to certain sexual orientations? So privacy only for some?
And privacy for what? To build what world with that privacy? That shouldn't be the only thing important in ethical computing. The other basic human rights should also be a goal. What is even privacy for then?

@AhmadDakhlallah

> We're punishing people for whatever they want to do in thier own instance

In this case, the “punition” is to not promote people’s tools that punish people. To not send people there knowing there are such discriminatory TOS. The operator isn’t punished.

> because you don't like how they think.

**they**(the instance operator) don't like how a lot of people are and people who are okay with that. That's the initial "don't like" and it's discriminatory. Then the reaction is to not give visibility (and implicit trust) to them.

> We force our freedom over others

What is this twisting of the situation? Since when not listing a discriminatory instance is forcing freedom over others? And yes, if was IRL for a shop, it would be fair to sue them for discrimination. There shouldn't be a rule of "not forcing freedom over others". Every discrimination is possible otherwise. Including your utility provider to cut your electrical power for your political views. And they shouldn't have that freedom. So yes freedom should be forced on the utility provider so they can't discriminate in the delivery of their service.

> If you disagree with the instance, don't use it.

And let's not recommend it by having it in the list. Invidious should have a policy about instance listing, and it shouldn’t limit itself to privacy.

> You have full freedom to do so.

Sure, freedom, but there is clear bias in favor this discrimination by letting it here. That's a fake neutrality to rely on the freedom to pick the instance and read every TOS. We are users of libreddit, we know it's bullshit to rely on the freedom to choose after looking at the TOS of every online service. That’s how people defend GAFAMs/big tech/surveillance capitalism.

> You're not accomplishing anything by hacking it and removing it.

What is this hacking thing? That's just removing it from a list here...
And that accomplish to have a list that is more trustable and doesn't push people to a problematic instance. Same as when flagging instances for cloudflare. That means having a policy here that care about some important things of the instances policies.

> I mean other than being an oppressor who is forcing his "freedom" by taking others freedom.

What a way to reverse the situation: Wanting to unlist an instance with oppressive TOS is oppression now. And it's totally not taking freedom from people to forbid a service based on sexual orientation. But unlisting an instance that do so is taking freedom? 🤷

> Would you do the same if the instance is gils only or gay only or doctors only or some other groups only people? 

Makes no sense for libreddit, a reddit proxy so… There doesn't seem to be any kind of non-mixed policy that would make sense here.
If there was, there would need to be accounted that there isn't a systemic discrimination against non-girls, non-gays, non-doctors. That's why it's not the same to have an only-palestinian or only-israelian group/service in Palestine (using the large definition that includes israel). It's not a mirror situation, we have to first look and acknowledge the current state of power struggle and eventual oppression in the society. It’s not about the numbers. There is more right-handed people, but there is little oppression against left-handed people. (at least in the countries that stopped doing that a few decades ago, I don’t know for others)
So for such hypothetical instances, a tag like for cloudflare at least to make it filterable because they aren’t general public instances. Then we would have to see the specific case to get the context of the public limitation of the TOS.

> This one is straight people only.

And it's the current dominant group in terms of power struggle and attacks on the excluded groups. So that goes in that direction of the current domination.

> you need to take down a lot more than this one. Many instances are specific to a group of people.

Really ? In their TOS they **forbib** access to a group of people or have it reserved **exclusively**  ?  And it includes Invidious ?
